light_curves/multiband_lc.ipynb

```diff
@@ -110,47 +91,39 @@
+    "    from alerce.core import Alerce\n",
     "\n",
     "try:\n",
     "    from unTimely_Catalog_tools import unTimelyCatalogExplorer\n",
     "except ImportError:\n",
-    "    if not os.path.exists('./unTimely_Catalog_explorer'):\n",
-    "        !git clone https://github.com/fkiwy/unTimely_Catalog_explorer.git\n",
-    "    sys.path.append('./unTimely_Catalog_explorer')\n",
+    "    !pip install git+https://github.com/fkiwy/unTimely_Catalog_explorer\n",
+    "    !pip install reproject\n",
     "    from unTimely_Catalog_tools import unTimelyCatalogExplorer\n",
-    "    \n",
-    "\n",
-    "!pip install alerce\n",
-    "\n",
     "\n",
-    "import tempfile\n",
-    "\n",
-    "# Local code imports\n",
+    "# local code imports\n",
     "sys.path.append('code/')\n",
-    "from fluxconversions import convert_WISEtoJanskies, convertACSmagtoflux\n",
-    "from panstarrs import panstarrs_get_lightcurves, ps1cone, ps1search, checklegal, ps1metadata, addfilter, improve_filter_format, search_lightcurve\n",
-    "from clean_filternames import clean_filternames\n",
-    "from gaia_functions import Gaia_get_lightcurve, Gaia_extract_median_photometry, Gaia_retrieve_median_photometry, Gaia_retrieve_EPOCH_PHOTOMETRY, Gaia_mk_lightcurves, Gaia_mk_MultiIndex, Gaia_plot_lightcurves\n",
-    "from HCV_functions import get_hscapiurl, hcvcone, hcvsearch, hcvmetadata, cat2url, checklegal_hcv, HCV_get_lightcurves\n",
-    "from mast_functions import resolve, mastQuery\n",
-    "from icecube_functions import icecube_get_lightcurve, icecube_get_catalog, icecube_download_data\n",
+    "from panstarrs import panstarrs_get_lightcurves\n",
+    "from gaia_functions import Gaia_get_lightcurve\n",
+    "from HCV_functions import HCV_get_lightcurves\n",
+    "from icecube_functions import icecube_get_lightcurve\n",
     "from sample_selection import get_lamassa_sample, get_macleod16_sample, get_ruan_sample, get_macleod19_sample, get_sheng_sample, \\\n",
     "    get_green_sample, get_lyu_sample, get_lopeznavas_sample, get_hon_sample, get_yang_sample,  get_SDSS_sample, remove_duplicate_coords\n",
-    "from ztf_functions import ZTF_id2coord,ZTF_get_lightcurve\n",
+    "from ztf_functions import ZTF_get_lightcurve\n",
     "from data_structures import MultiIndexDFObject\n",
     "from heasarc_functions import HEASARC_get_lightcurves\n",
     "from TESS_Kepler_functions import TESS_Kepler_get_lightcurves\n",
     "from WISE_functions import WISE_get_lightcurves\n",
-    "from plot_settings import setup_text_plots\n",
+    "\n",
     "\n",
     "## NEW! LOAD/SAVE PICKLE FILE! ##\n",
     "SAVEDF = False # if set to True, pickle file will be saved. If False, pickle file will be loaded.\n",
     "#pickle_file_name = \"data/dflc_Shooby_March102023.pkl\" # name of pickle file (to be loaded or saved)\n",
     "pickle_file_name = \"data/dflc_Krick_Feb232023.pkl\" # name of pickle file (to be loaded or saved)\n",
-    "###\n",
-    "\n"
+    "###"
```

Comment:
This should go when the notebook is ready to be shipped, but until then, please move to a separate cell.

Reply:
thanks, done